api/modules/dataSyncs.js

```diff
@@ -131,6 +146,12 @@ const dataSyncs = module.exports = (() => {
                         }
                     })
                 }
+                if (i.user) {
+                    await syncContributions(i.html_url, i.user.html_url, 1, 0, matchingIssue, i)
```

Comment:
See https://github.com/setlife-network/trinary/pull/543/files#r771479769

It is a lot harder to understand what each of these parameters is for without having to look up the function

If we use an object with property names it improves code readability